src/playlist-loader.js

```diff
@@ -496,6 +504,18 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
 
       loader.state = 'HAVE_MASTER';
 
+      if (req.responseURL) {
+        // checking, whether manifest request ended with redirect
```

Comment:
Maybe we can reuse the comment from above, just worded a little differently for this case (just for consistency).